test/conformance/ptp/prometheus.go

```diff
@@ -0,0 +1,133 @@
+package ptp
```

Comment:
What does this file do?

Reply:
This supports one of the test imported by cnf-features-deploy. I get the metrics from ptp anf check if they are in prometheus locally